python/ray/dag/compiled_dag_node.py

```diff
@@ -415,13 +417,15 @@ class DAGDriverProxyActor:
 
     def __init__(
         self,
-        buffer_size_bytes: Optional[int],
+        execution_timeout: Optional[float] = DEFAULT_ADAG_PUT_TIMEOUT_S,
```

Comment:
Can you follow the same convention used by other parameters here, as introduced by #46268? I.e. set to None here and then take it from the context.